omnibus/config/software/cleanup.rb

```diff
@@ -0,0 +1,8 @@
+name 'cleanup'
+default_version '1.0.0'
+skip_transitive_dependency_licensing true
+license :project_license
+
+build do
+  delete "#{install_dir}/embedded/nodejs"
```

Comment:
Will this also clean up the downloaded npm packages that are used at build time?